test/sendReceive.spec.ts

```diff
@@ -30,7 +30,7 @@ const testMessages: SendableMessageInfo[] = [
   }
 ];
 
-function testReceivedMessages(receivedMsgs: ServiceBusMessage[]) {
+async function testReceivedMessages(receivedMsgs: ServiceBusMessage[]): Promise<void> {
```

Comment:
Everything inside `testReceivedMessages` is synchronous. Why make it `async`?

Reply:
I was resolving linter errors. I will remove async.